packages/typescript/tsconfig.prod.json

```diff
@@ -1,6 +1,3 @@
 {
-  "extends": "./tsconfig.json",
-  "compilerOptions": {
-    "outDir": "./dist"
```

Comment:
`outDir` config should live on the user package config. Otherwise, the built files will end up on the `config-typescript` package.